.gitlab/pipeline/zombienet/bridges.yml

```diff
@@ -9,9 +9,9 @@
   rules:
     - !reference [.build-refs, rules]
     - if: $CI_COMMIT_REF_NAME =~ /^gh-readonly-queue.*$/
```

Comment:
I think we should delete this if the variable assignment has been moved.

Reply:
I think we should keep, since this rule make these test _only_ run on merge queues.

Reply:
It should also run in the CI. And it seems to be running. If this rule makes it only run in merge queues it's by mistake.